Sources/PrimerSDK/Classes/Data Models/PaymentMethodTokenizationRequest.swift

```diff
@@ -34,6 +34,10 @@ struct BankSelectorTokenizationRequest: TokenizationRequest {
     let paymentInstrument: PaymentInstrument
 }
 
+struct BlikPaymentMethodTokenizationRequest: TokenizationRequest {
+    let paymentInstrument: BlikPaymentMethodOptions
+}
```

Comment:
Are we going with implicit `internal`s ? I'm fine with them - just wanted to get some inputs in case something has chanced in our coding style 👍